reana_snakemake.yaml

```diff
@@ -0,0 +1,13 @@
+version: 0.8.0
```

Comment:
Please change the file name to the usual `reana-snakemake.yaml` (i.e. dash instead of underscore).